### PR TITLE
SDR Folio downtime management

### PR DESCRIPTION
**Pull request by lwrubel**
Describe steps for handling FOLIO full or partial downtime on the SDR/Infra side of things.

### CODE REVIEW

**Review by mjgiarlo — APPROVED** [review]
✨ 

Thank you for tracking this all down and documenting it so clearly!

File: best-practices/folio_downtime.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  * check-marc: cron job to check whether cataloger has fully cataloged ETD. This can still run while rollover happens, because all it does is look for is a status change. It does not change any FOLIO record.
+  * catalog-status: once the ETD is cataloged fully, the ETD can be accessioned and released to SW. This WF is OK to run since it does not change FOLIO records.
+* releaseWF
+  * releaseWF_update-marc_dsa: adds/updates MARC 856 links to purl
+
+hydra_etd has a submit_marc queue for CreateStubMarcRecordJob, with its own process. 
+
+
+### Before the rollover
+* Quiet the relevant processes which monitor queues for jobs that update FOLIO. Quieting queues means that they finish any in-flight jobs and don't pick up any more jobs. Use the Sidekiq UI to quiet the queue by using the "Pause" button.
```
Comment (jmartin-sul) [inline best-practices/folio_downtime.md:50]: "Pause" -> "Quiet"?

**Review by jmartin-sul — APPROVED** [review]
approve pending consideration of one small nitpick

File: best-practices/folio_downtime.md (hunk 1 above)
Comment (lwrubel) [reply best-practices/folio_downtime.md]: Thanks, fixed. We also get a "Pause" button with Sidekiq pro, but let's use the same process across Sidekiqs.

Comment (jmartin-sul) [reply best-practices/folio_downtime.md]: ah, sorry, didn't realize that, good to know

Comment (jmartin-sul) [reply best-practices/folio_downtime.md]: but yeah, also, yay consistency

**Pull request merged**